editor/gui/editor_run_bar.cpp

```diff
@@ -465,25 +503,55 @@ EditorRunBar::EditorRunBar() {
 	main_hbox->add_child(run_native);
 	run_native->connect("native_run", callable_mp(this, &EditorRunBar::_run_native));
 
-	play_scene_button = memnew(Button);
+	bool add_play_xr_mode_options = false;
+#ifndef _3D_DISABLED
+	if (OS::get_singleton()->has_feature("xr_editor") &&
+			(XRServer::get_xr_mode() == XRServer::XRMODE_ON ||
+					(XRServer::get_xr_mode() == XRServer::XRMODE_DEFAULT && GLOBAL_GET("xr/openxr/enabled")))) {
+		// If this is the XR editor and openxr is enabled, we turn the `play_scene_button` and
+		// `play_custom_scene_button` into MenuButtons to provide the option to start a scene in
+		// either regular mode or XR mode.
+		add_play_xr_mode_options = true;
+	}
+#endif // _3D_DISABLED
+
+	if (add_play_xr_mode_options) {
+		MenuButton *menu_button = memnew(MenuButton);
+		PopupMenu *popup = menu_button->get_popup();
+		popup->add_item(TTRC("Run Current Scene in Regular mode"), 0);
+		popup->add_item(TTRC("Run Current Scene in XR mode"), 1);
+		popup->connect(SceneStringName(id_pressed), callable_mp(this, &EditorRunBar::_play_current_pressed));
+		play_scene_button = menu_button;
+	} else {
+		play_scene_button = memnew(Button);
+		play_scene_button->set_toggle_mode(true);
+		play_scene_button->connect(SceneStringName(pressed), callable_mp(this, &EditorRunBar::_play_current_pressed).bind(-1));
+	}
 	main_hbox->add_child(play_scene_button);
 	play_scene_button->set_theme_type_variation("RunBarButton");
-	play_scene_button->set_toggle_mode(true);
 	play_scene_button->set_focus_mode(Control::FOCUS_NONE);
 	play_scene_button->set_tooltip_text(TTRC("Run the currently edited scene."));
-	play_scene_button->connect(SceneStringName(pressed), callable_mp(this, &EditorRunBar::_play_current_pressed));
 
 	ED_SHORTCUT_AND_COMMAND("editor/run_current_scene", TTRC("Run Current Scene"), Key::F6);
 	ED_SHORTCUT_OVERRIDE("editor/run_current_scene", "macos", KeyModifierMask::META | Key::R);
 	play_scene_button->set_shortcut(ED_GET_SHORTCUT("editor/run_current_scene"));
 
-	play_custom_scene_button = memnew(Button);
+	if (add_play_xr_mode_options) {
+		MenuButton *menu_button = memnew(MenuButton);
+		PopupMenu *popup = menu_button->get_popup();
+		popup->add_item(TTRC("Run Specific Scene in Regular mode"), 0);
+		popup->add_item(TTRC("Run Specific Scene in XR mode"), 1);
```

Comment:
```suggestion
		popup->add_item(TTRC("Run in Regular mode"), 0);
		popup->add_item(TTRC("Run in XR mode"), 1);
```
Since the user will be prompted with a dialog to select a scene, I think it's unnecessary to include "specific scene" in the label.